refinery/ui/source/js/file-browser/partials/input-groups-column-template.html

```diff
@@ -0,0 +1,33 @@
+<div class="ngCellText grid-input-groups">
+  <div
+    ng-if="grid.appScope.nodeSelectCollection[row.entity.uuid].groupList.length > 0"
```

Comment:
Would it makes sense to factor out `grid.appScope.nodeSelectCollection[row.entity.uuid]`? It is used several times just in this section.